get-debug-perm.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+function getToken() {
+    # Create a secret to hold a token for the "tekton-results-debug" service account
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: default-token
+  namespace: tekton-pipelines
+  annotations:
+    kubernetes.io/service-account.name: tekton-results-debug
+type: kubernetes.io/service-account-token
+EOF
+
+    # Wait for the token controller to populate the secret with a token:
+    while ! kubectl describe secret default-token -n tekton-pipelines | grep -E '^token' >/dev/null; do
+    echo "waiting for token..." >&2
+    sleep 1
+    done
+
+    # Get the token value
+    $(kubectl get secret default-token -n tekton-pipelines -o jsonpath='{.data.token}' | base64 --decode > /tmp/token)
+
+    TOKEN=$(cat /tmp/token)
+}
+
+export KO_DOCKER_REPO="kind.local"
```

Comment:
Why are we setting a ko environment variable here? This is only needed if a developer is building the Results container images.